<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="refresh_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refrescar base de datos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stats_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gr√°ficos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ettiquette_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Etiquetado</string>
    </property>
   </widget>
   <widget class="QSplitter" name="pathwaysanswer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>191</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="xlsx_route_label">
     <property name="text">
      <string>Ruta archivo original:</string>
     </property>
    </widget>
    <widget class="QLabel" name="xlsx_route_response_label">
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="route_destiny_label">
     <property name="text">
      <string>Ruta destino:</string>
     </property>
    </widget>
    <widget class="QLabel" name="route_destiny_response_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="darwinizer_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Darwinizer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Archivos</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionDestino"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionDestino">
   <property name="text">
    <string>Destino</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
